src/multiply_kernel_ompGemm_m.f90

```diff
@@ -390,8 +345,8 @@ subroutine m_kern_min(k_off, kpart, ib_nd_acc, ibaddr, nbnab,       &
     integer :: kpart, k_off
     ! Remember that a is a local transpose
     real(double) :: a(lena)
-    real(double) :: b(lenb)
-    real(double) :: c(lenc)
+    real(double), target :: b(lenb)
```

Comment:
Why are these targets?

Reply:
These are left-overs from the previous version where I was using pointers. Thanks for spotting, I'll remove them.